### PR TITLE
ci: remove material-unit-tests failure blocklist

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits

### CODE REVIEW

File: docs/DEBUG_MATERIAL_IVY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The `yarn test` script from the `components` repository runs all tests in the project.
+This is sometimes not desired because it involves building and testing of all packages
+and entry-points. Running tests for an individual entry-point is possible by explicitly
+selecting a given test target.
+
+Here is an example of commands that run individual test targets. Note that it is
+**important** to specify the `--define=compile=aot` flag in order to run tests with Ivy.
+  
+```bash
+yarn bazel --define=compile=aot src/material/slider:unit_tests
```
Comment (JoostK) [inline docs/DEBUG_MATERIAL_IVY.md]: Shouldn't these be

```suggestion
yarn bazel test --define=compile=aot src/material/slider:unit_tests
```

Comment (devversion) [reply docs/DEBUG_MATERIAL_IVY.md]: Ops yeah. Fixed :smile:

**Review by gkalpak — APPROVED** [review]
🚀

File: docs/DEBUG_MATERIAL_IVY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Debugging the Material unit tests job
+
+Currently all changes to Ivy are validated against the test suite of the Angular
+components repository. In order to debug the `material-unit-tests` CI job, the
```
Comment (kara) [inline docs/DEBUG_MATERIAL_IVY.md]: Nit: might help to be more specific about where it lives

```suggestion
@ angular/components repository. In order to debug the `material-unit-tests` CI job, the
```

File: docs/DEBUG_MATERIAL_IVY.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+following steps can be used:
+
+1\) Build the Ivy package output by running `./scripts/build-ivy-npm-packages.sh`.
+2\) Set up the package output in the `components` repository by running:
```
Comment (kara) [inline docs/DEBUG_MATERIAL_IVY.md]: When I first read this, I thought you meant to run it in the components repository. Can we add:

```suggestion
2\) Set up the package output in the `components` repository by running (in this repo):
```

Comment (kara) [inline docs/DEBUG_MATERIAL_IVY.md]: Can we also add a step before this to clone down the repo if you don't already have it?

File: docs/DEBUG_MATERIAL_IVY.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+### Running tests for individual entry-points
+
+The `yarn test` script from the `components` repository runs all tests in the project.
+This is sometimes not desired because it involves building and testing of all packages
+and entry-points. Running tests for an individual entry-point is possible by explicitly
+selecting a given test target.
+
+Here is an example of commands that run individual test targets. Note that it is
+**important** to specify the `--define=compile=aot` flag in order to run tests with Ivy.
```
Comment (kara) [inline docs/DEBUG_MATERIAL_IVY.md]: 👍 This is super helpful

**Review by kara — APPROVED** [review]
LGTM, a few things

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: global approval

**Comment by devversion** [comment]
Closing this in favor of a new PR.

**Pull request closed without merging**